lib/queue-factory.ts

```diff
@@ -102,7 +104,16 @@ export function getRedisClient(
 ) {
   if (!redisClients[type]) {
     if (clusterNodes && clusterNodes.length) {
-      redisClients[type] = new Redis.Cluster(clusterNodes, redisOpts);
+      redisClients[type] = new Redis.Cluster(clusterNodes, {
+        ...redisOpts, 
+        redisOptions: {
```

Comment:
No it won't. Passing undefined is like not passing anything there. It will also not work without these options as far as I know. Redis Cluster in IOREDIS is weird. Took me long time to get it to connect with ScaleWay. 

Let me try getting these params out from URL from first node. Passing this feels redundant.